patches/server/0929-Detect-headless-JREs.patch

```diff
@@ -0,0 +1,56 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Noah van der Aa <ndvdaa@gmail.com>
+Date: Sat, 22 Oct 2022 14:47:45 +0200
+Subject: [PATCH] Detect headless JREs
+
+Crashes caused by the missing AWT dependency come up in the support channels fairly often.
+This patch detects the missing dependency and stops the server with a clear error message,
+containing a link to instructions on how to install a non-headless JRE.
+
+diff --git a/src/main/java/io/papermc/paper/util/ServerEnvironment.java b/src/main/java/io/papermc/paper/util/ServerEnvironment.java
+index 6bd0afddbcc461149dfe9a5c7a86fff6ea13a5f1..680f9c22972aaaa01e55fe351396823f78248a7a 100644
+--- a/src/main/java/io/papermc/paper/util/ServerEnvironment.java
++++ b/src/main/java/io/papermc/paper/util/ServerEnvironment.java
+@@ -4,6 +4,7 @@ import com.sun.security.auth.module.NTSystem;
+ import com.sun.security.auth.module.UnixSystem;
+ import org.apache.commons.lang.SystemUtils;
+ 
++import java.awt.Color;
```

Comment:
import?

Reply:
Inlined, tho it shouldn't really matter conflict wise since it's a Paper class